_includes/head-idr.html

```diff
@@ -41,6 +41,7 @@
                         <ul class="submenu menu vertical" data-submenu>
                             <li><a href="{{ "/index.html" | prepend: site.baseurl }}">Overview</a></li>
                             <li><a href="{{ "/studies.html" | prepend: site.baseurl }}">Published studies</a></li>
+                            <li><a href="{{ "/ontologies.html" | prepend: site.baseurl }}">Linked resources</a></li>
```

Comment:
I'm not a huge fan of the long "/linked-resources" in the URL, but it might be better than just "ontologies"

Reply:
I was thinking the same. I will change the name of the file so it is clearer

Reply:
a compromise will be to use ``resources.html``

Reply:
yeah, but there's some room for confusion since we try to avoid "repositories" by using "resources".